modules/auxiliary/gather/http_pdf_authors.rb

```diff
@@ -0,0 +1,183 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'pdf-reader'
+
+class MetasploitModule < Msf::Auxiliary
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Gather PDF Authors',
+      'Description' => %q{
+        This module downloads PDF files and extracts the author's
+        name from the document metadata.
+      },
+      'License'     => MSF_LICENSE,
+      'Author'      => 'Brendan Coles <bcoles[at]gmail.com>'))
+    register_options(
+      [
+        OptString.new('URL', [ false, 'The URL of a PDF to analyse', '' ]),
+        OptString.new('URL_LIST', [ false, 'File containing a list of PDF URLs to analyze', '' ]),
```

Comment:
@bcoles As someone who was not aware of the `file:`, the URL option is much appreciated, as is the documentation. Well done!